neurons/model_scoring.py

```diff
@@ -107,7 +119,16 @@ def embed_text(imagebind, texts: List[str], device) -> List[torch.FloatTensor]:
 def get_model_score(hf_repo_id, mini_batch, local_dir, hotkey, block, model_tracker):
     cleanup_gpu_memory()
     log_gpu_memory('before model load')
-    inference_recipe, config = load_ckpt_from_hf(hf_repo_id, local_dir)
+    inference_recipe, config, hotkey_file_contents = load_ckpt_from_hf(hf_repo_id, local_dir)
+
+    # Check if the contents of license file are the same as the hotkey if in repo
+    if hotkey_file_contents is not None and hotkey_file_contents != hotkey:
```

Comment:
shouldn't this be `if hotkey_file_contents is None or hotkey_file_contents != hotkey` so that if it's None we don't just give it a free pass?